app/api/events.py

```diff
@@ -64,23 +111,30 @@ class Meta:
                            many=True,
                            type_='sponsor')
 
+    user = Relationship(attribute='user',
+                        self_view='v1.event_user',
+                        self_view_kwargs={'id': '<id>'},
+                        related_view='v1.user_detail',
+                        related_view_kwargs={'id': '<id>'},
+                        schema='UserSchema',
+                        type_='user')
+
 
 class EventList(ResourceList):
-    decorators = (jwt_required, )
-    schema = EventSchema
-    data_layer = {'session': db.session,
-                  'model': Event}
 
+    def before_create_object(self, data, view_kwargs):
+        data['starts_at'] = data['starts_at'].replace(tzinfo=None)
+        data['ends_at'] = data['ends_at'].replace(tzinfo=None)
```

Comment:
@LuD1161 I don't think this fixes the timezone aware issue. I checked, it still gives UTC time +00.00 even when the timezone is different